brainiak/reprsimil/brsa.py

```diff
@@ -421,7 +830,7 @@ def _prepare_DF(self, n_T, scan_onsets=None):
             D = self._D_gen(n_T)
```

Comment:
It is fine to flip the names. But it requires flipping all parameters related to these (XTDX and XTFX, etc.). I think doing this in a separate PR can help make sure that all variables are changed correctly, because this PR already has too many change to the code and may be hard to track these changes.